dev/cognitiveservices/data-plane/Language/common.json

```diff
@@ -320,6 +320,24 @@
         }
       }
     },
+    "RequestWarning": {
```

Comment:
How does this differ from InputWarning on L341 below?

Reply:
I have the same question.